src/auth/src/x509/authenticator.rs

```diff
@@ -51,10 +54,15 @@ impl X509Authenticator {
         tcp_stream: &TcpStream,
         authorization_request: AuthRequest,
     ) -> Result<bool, IoError> {
+        #[cfg(unix)]
```

Comment:
use [connection_fd](https://github.com/infinyon/future-aio/blob/master/src/net/mod.rs)

Reply:
I think this will make it cleaner. https://github.com/infinyon/future-aio/pull/80/files#diff-4c6a8ac4ba774ab01f57abae00bd7caef7c65b5bf09404c3f9a68b93fa06843bR60-R73

Reply:
So may be we should get PR80 on `future-aio` merge first?